packages/salesforcedx-vscode-core/README.md

```diff
@@ -71,6 +71,18 @@ After you run Apex tests, two new commands are available in the command palette:
 
 To retrieve code coverage results when you run Apex tests, edit your workspace settings and set `salesforcedx-vscode-core.retrieve-test-code-coverage` to `true`.  
 
+## Explore Your Apex Tests
+
+The Apex Tests sidebar provides several features. Here, you can see all your Apex tests at a glance. You can run one test method, the test methods in one class, or all your tests. You can view the results of your last test run. And, you can jump from those results to the corresponding lines in your code. To access this sidebar, click the flask icon (hover text: Test) in the view bar on the left side of the VS Code window. (If you don’t see this icon, make sure that the project you have open in VS Code contains an `sfdx-project.json` file in its root directory.)  
```

Comment:
Just an FYI, the icon seems to be officially called a beaker (https://octicons.github.com/icon/beaker/). I don't mind if we use flask or beaker but just wanted to point out the official icon name.